demos/security_barrier_camera_demo/README.md

```diff
@@ -2,7 +2,8 @@
 
 This demo showcases Vehicle and License Plate Detection network followed by the Vehicle Attributes Recognition and License Plate Recognition networks applied on top
 of the detection results. You can use a set of the following pre-trained models with the demo:
-* `vehicle-license-plate-detection-barrier-0106`, which is a primary detection network to find the vehicles and license plates
+* `vehicle-license-plate-detection-barrier-0106` or
+* `vehicle-license-plate-detection-barrier-0123`, which are primary detection networks to find the vehicles and license plates
```

Comment:
I suggest placing them on a single line like `license-plate-recognition-barrier-0001` and`license-plate-recognition-barrier-0007`

Reply:
Ok